Week6/app/src/main/java/com/oliverperez/blockbusterchallenge/model/RoomRepository.kt

```diff
@@ -1,42 +1,19 @@
 package com.oliverperez.blockbusterchallenge.model
 
-import android.os.AsyncTask
-import androidx.lifecycle.LiveData
 import com.oliverperez.blockbusterchallenge.app.BlockbusterApplication
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
 
-class RoomRepository :
-    MovieRepository {
+class RoomRepository(private val scope: CoroutineScope) : MovieRepository {
```

Comment:
You don't have to pass in scope here. 
Simply put `suspend` modifier on every method in the repository and launch the coroutine in the ViewModel